pulsar-client-cpp/lib/ConnectionPool.cc

```diff
@@ -80,4 +80,20 @@ Future<Result, ClientConnectionWeakPtr> ConnectionPool::getConnectionAsync(
     return future;
 }
 
+void ConnectionPool::close() {
```

Comment:
Could this be done implicitly in the destructor itself?

Reply:
Yes. I added the destructor instead of the `close()` method.